mistralrs-core/src/pipeline/mod.rs

```diff
@@ -56,12 +56,12 @@ pub trait ModelPaths {
     /// Model weights files (multiple files supported).
     fn get_weight_filenames(&self) -> &[PathBuf];
 
-    /// Retrieve the PretrainedConfig file.
-    /// See: https://huggingface.co/docs/transformers/v4.40.2/en/main_classes/configuration#transformers.PretrainedConfig
+    /// Retrieve the [PretrainedConfig] file.
+    /// [PretrainedConfig]: https://huggingface.co/docs/transformers/v4.40.2/en/main_classes/configuration#transformers.PretrainedConfig
```

Comment:
Should this be backtick wrapped like other values?

```suggestion
    /// Retrieve the [`PretrainedConfig`] file.
    /// [`PretrainedConfig`]: https://huggingface.co/docs/transformers/v4.40.2/en/main_classes/configuration#transformers.PretrainedConfig
```